pkg/skaffold/instrumentation/meter.go

```diff
@@ -97,3 +105,29 @@ func AddDevIterationErr(i int, errorCode proto.StatusCode) {
 func AddFlag(flag *flag.Flag) {
 	meter.EnumFlags[flag.Name] = flag
 }
+
+func ExportMetrics(exitCode int) error {
+	home, _ := homedir.Dir()
```

Comment:
I doubt this will return an error often (if ever), but should we still handle the error anyway?

Reply:
Yeah makes sense to do so. Done!